TESTS/mbed_drivers/watchdog/main.cpp

```diff
@@ -225,7 +225,7 @@ void test_start_max_timeout()
     Watchdog &watchdog = Watchdog::get_instance();
     uint32_t max_timeout = watchdog.get_max_timeout();
     TEST_ASSERT_TRUE(watchdog.start(max_timeout));
-    // The watchdog should trigger at, or after the timeout value.
+    // The watchdog should trigger at, or after the timeout value;
```

Comment:
this change is not related. the rest of the comments in this file has it as it was